proof_of_concept/definitions/identifier.py

```diff
@@ -0,0 +1,101 @@
+"""Identifier for various things in the DDM."""
+import re
+from typing import Any, cast, List, Type
+
+
+class Identifier(str):
```

Comment:
Maybe you can refer to `components` instead of `parts` to match the documentation of terminology.

Reply:
Yeah. The issue is that "component" is already a package, and I don't like overloading terms. I tried changing the documentation instead, replacing "component" with "part" there, but that reads a bit weird. I'm thinking to try "section" now.

Reply:
Fair enough. Maybe `splits` or `elements`? Or `section` is also fine I guess.

Reply:
Pieces, parts, fragments, shards, sections or components.
A segment or a unit or an item?
The hardest task in programming
is not solved in Python.

Reply:
Segments it is, it kind of looks like a worm anyway :-).

Reply:
Lol! Are you in sinterklaas mode?